frontend/pages/hosts/components/DeleteHostModal/DeleteHostModal.tsx

```diff
@@ -59,12 +58,7 @@ const DeleteHostModal = ({
   };
 
   return (
-    <Modal
-      title="Delete host"
-      onExit={onCancel}
-      onEnter={onSubmit}
-      className={baseClass}
-    >
+    <Modal title="Delete host" onExit={onCancel} className={baseClass}>
```

Comment:
I'm going to follow up with a modals PR since we have issues with highlighting the Cancel button and enter still submitting. I think that's why I removed it here but still want to fix globally where on enter does submit except when Cancel or X are in focus